index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+    	<link rel="stylesheet" type="text/css" href="css/style.css">
+		<title>JQuery</title>
+	</head>
+	<body>
+		<div>
+			<div>
+				<p><span>20 amazing Indian Ocean holidays</span></p>
+			</div>
+			<div>
+				<p>Mauritius</p>
+			</div>
+			<div>
+				<p><span>Sri Lanka</span></p>
+			</div>
+			<div>
+				<p>Maldives</p>
+			</div>
+			<div>
+				<p><span>Mauritius</span></p>
+			</div>
+			<div>
+				<p>Sri Lanka</p>
+			</div>
+			<div>
+				<p><span>Maldives</span></p>
+			</div>
+			<div>
+				<p>Mauritius</p>
+			</div>
+			<div>
+				<p><span>Sri Lanka</span></p>
+			</div>
+			<div>
+				<p>Maldives</p>
+			</div>
+		</div>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script><script>window.jQuery || document.write('<script src="js/vendor/jquery-3.2.1.min.js"><\/script>')</script>
```

Comment:
tutaj wystarczy tylko pierwszy tag script ladujacy jQuery, chyba, ze faktycznie chcesz miec pewnosc, ze jQuery bedzie zawsze zaladowane (czyli jesli nie bedzie dostepna z adresu ajax.googleapis.com to zaladuj z katalogu), to musialabys dodac te biblioteke w lokalizacji jaka podajesz, czyli: `js/vendor/jquery-3.2.1.min.js`